app/views/posts/new.html.erb

```diff
@@ -1,30 +1,40 @@
-<h1 class="has-margin-bottom-2">
-  New Post
+<% unless @category.present? && @category.min_trust_level.present? && @category.min_trust_level > current_user.trust_level %>
+  <h1 class="has-margin-bottom-2">
+    New Post
+    <% if @category.present? %>
+      in <%= @category.name %>
+    <% end %>
+  </h1>
   <% if @category.present? %>
-    in <%= @category.name %>
+    <p class="has-color-tertiary-500">Not where you meant to post? See <%= link_to 'Categories', categories_path %></p>
+
+    <div class="notice is-info">
+      <p><strong>Posting Tips</strong></p>
+      <div class="has-font-size-caption">
+        <% guidance = @category.asking_guidance_override %>
+        <%= raw(sanitize(render_markdown(guidance.present? ? guidance : SiteSetting['AskingGuidance']), scrubber: scrubber)) %>
+      </div>
+    </div>
   <% end %>
-</h1>
-<% if @category.present? %>
-  <p class="has-color-tertiary-500">Not where you meant to post? See <%= link_to 'Categories', categories_path %></p>
 
-  <div class="notice is-info">
-    <p><strong>Posting Tips</strong></p>
-    <div class="has-font-size-caption">
-      <% guidance = @category.asking_guidance_override %>
-      <%= raw(sanitize(render_markdown(guidance.present? ? guidance : SiteSetting['AskingGuidance']), scrubber: scrubber)) %>
+  <% if @post.errors.any? %>
+    <div class="notice is-danger is-filled">
+      <p>The following errors prevented your post from being saved:</p>
+      <ul>
+        <% @post.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+      </ul>
     </div>
-  </div>
-<% end %>
+  <% end %>
 
-<% if @post.errors.any? %>
+  <%= render 'form', post: @post, post_type: @post_type, category: @category, submit_path: request.path %>
+<% else %>
   <div class="notice is-danger is-filled">
```

Comment:
We don't really need the lead text and bullet points here - this whole `<div>` can just be a single error message:

```erb
<div class="notice is-danger is-filled">
  You don't have a high enough trust level to post in the <%= @category.name %> category.
</div>
```

Reply:
How can a user know that they don’t have high enough trust level for posting in category without a error message? I was thinking that tips of posting in blogs is part of form so, it shouldn’t show for low trust level users. 🤔 😴